assets/styles/components/talk-to-us-text-box.scss

```diff
@@ -0,0 +1,3 @@
+.govuk-talk-to-us{
+    padding:18px; background:#f8f8f8cc; font-family: Helvetica, Arial, sans-serif; font-size: 12px; line-height: 23px
+}
```

Comment:
These styles should follow the same convention as the rest of the CSS. E.g.

```
.example-selector,
.second-selector {
    clear: both;
    display: block;
    padding: 1rem 2rem;
}
```

The rules in general are:

 - Each property value pair should be on a new line
 - Each property value pair should end with a semi-colon, regardless of whether any others follow
 - Always have a space after colons
 - Properties should be alphabetised for easier scanning (see my example above)
 - Selectors should occupy a line each
 - There should always be a space before the opening bracket

I'll see if I can find a formal write up of these rules for you.

I don't think this shoudl necessarily block the pull request. Although I would strongly recommend you follow these guidelines as they make CSS much easier to read, understand, and modify.

In general having everything on a line like this greatly increases the chance of merge conficts and make git diffs harder to read (and thus less useful).